polygon/rest/models/aggs.py

```diff
@@ -4,24 +4,105 @@
 
 @dataclass
 class Agg:
+    open: float
+    high: float
+    low: float
+    close: float
+    volume: float
+    vwap: float
     timestamp: int
+    transactions: int
+
```

Comment:
Zack showed me that you can make this concise by doing
`@staticmethod
    def from_dict(d):
        return Agg(**d)`
which works when the return attributes are longer than one character (like in this case). NBD, just wanted to say in case you prefer it that way.